src/main/java/org/checkerframework/checker/https/HTTPSAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,55 @@
+package org.checkerframework.checker.https;
+import com.sun.source.tree.*;
```

Comment:
It's good style to leave a blank line between the package and import statements.